.github/workflows/test-application.yaml

```diff
@@ -53,9 +76,18 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  composer validate --strict
+                  composer validate
                   composer update ${{ matrix.composer-flags }}
 
+            - name: Lint
+              if: ${{ matrix.lint }}
+              run: |
+                  Tests/Application/bin/adminconsole cache:clear --env dev
+                  Tests/Application/bin/websiteconsole cache:clear --env dev
+                  composer lint
+              env: ${{ matrix.env }}
+
```

Comment:
If there is no `lint-container` script defined in the `composer.json`, add one

Reply:
Like this https://github.com/sulu/SuluArticleBundle/blob/master/.github/workflows/test-application.yaml#L169-L180